src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs

```diff
@@ -444,25 +444,25 @@ internal static string DecodeStream(Stream stream, ref Encoding encoding)
                 isDefaultEncoding = true;
             }
 
-            string content = StreamToString (stream, encoding);
+            string content = StreamToString(stream, encoding);
             if (isDefaultEncoding) do
-            {
```

Comment:
This formatting change looks odd to me.  Would it be better to wrap the `do..while` in braces?

```c#
if (...)
{
  do
  {
  }
  while ()
}
```

Reply:
CodeFormatter keeps previous formatting for `if-do` but remove extra braces in `do-while`.
So formally you ask other change - manual vs automatic formatting.

I agree that you request is right and I'll manually do the fix.